mlflow/utils/validation.py

```diff
@@ -17,6 +17,14 @@
     " spaces ( ), and slashes (/)."
 )
 
+MAX_PARAMS_TAGS_PER_BATCH = 100
+MAX_METRICS_PER_BATCH = 1000
+MAX_ENTITIES_PER_BATCH = 1000
+MAX_BATCH_LOG_REQUEST_SIZE = int(1e7)
+MAX_PARAM_LENGTH = 500
```

Comment:
Q: Do we want to add metric/param/tag length validations in OSS? For context, we currently validate metric/param/tag names in the user-facing Python SDK (fluent & client APIs) as well as in the backing store implementations (SqlAlchemyStore, FileStore). However, length limits are currently defined per-store (i.e. the SQLAlchemyStore restricts metric names to 250 chars, while the FileStore imposes no limit but will fail to write metrics if their names exceed the filesystem's max filename length). It seems most consistent to add length validations at the store & client level as well so that the behavior/validity of user code isn't dependent (or is minimally dependent) on what store they're running against. However we could also consider adding such validation in a follow-up & applying it to all store APIs.

Reply:
I'll actually remove all validations from this PR for now and add them in a follow-up